src/04-modality-specific-files/10-functional-near-infrared-spectroscopy.md

```diff
@@ -0,0 +1,367 @@
+loss of crucial metadata specific to manufacturers and specific fNIRS systems.
+We also encourage users to provide additional meta information extracted from
+the manufacturer specific data files in the sidecar JSON file. This allows for
+easy searching and indexing of key metadata elements without needing to parse
+the various proprietary (typically binary) native data files. Other relevant
+files should  be included alongside the fNIRS data.
+
+### Terminology
+
+For proper documentation of fNIRS recording metadata it is important
+to understand the difference between a source, detector, and channel as these
+are defined differently to other modalities such as EEG. The following definitions
+apply in this document:
+
+-   Source - A light emitting device, sometimes called a transmitter.
+
+-   Detector - A photoelectric transducer, sometimes called a receiver.
+
+-   Channel - A paired coupling of a source and a detector with one specific light wavelength.
+    It is common for a single Source-Detector pair to result in two or more channels
+    with different wavelengths.
+
+### Sidecar JSON (`*_nirs.json`)
+
+It is common within the fNIRS community for researchers to build their own caps
+and optode holders to position their sources and detectors, or for optodes to
+be directly attached to the scalp with adhesive. To facilitate description of
+the wide variety of possible configurations several fields are RECOMMENDED within
+the `_*nirs.json` file.
+To clarify the usage and interaction of these fields the following examples are provided.
+If a commercial cap such as EasyCap actiCAP 64 Ch Standard-2
+was used then the values `CapManufacturer = ”EasyCap”`, `CapManufacturersModelName = ”actiCAP
+64 Ch Standard-2”` and `NIRSPlacementScheme = ”n/a”` should be used.
+If an EasyCap was used but with custom positions,
+as may be done by cutting custom holes in the cap,
+then the values `CapManufacturer = ”EasyCap”`, `CapManufacturersModelName = ”custom”` and
+`NIRSPlacementScheme = ”n/a”` should be used.
+If a completely custom cap was knitted then
+`CapManufacturer = ”custom”`, `CapManufacturersModelName = ”custom”` and
+`NIRSPlacementScheme = ”n/a”`.
+If no cap was used and optodes were taped to the scalp
+at positions Cz, C1 and C2, then the values `CapManufacturer = ”none”`, `CapManufacturersModelName
+= ”n/a”` and `NIRSPlacementScheme = ”["Cz", "C1", “C2]”` should be used.
+
+Closely spaced source detector pairs are often included in fNIRS measurements to
+obtain a measure of systemic, rather than neural, activity. These source-detector
+pairs are referred to as short channels. There is variation in how manufacturers
+implement these short channels, some use specialised sources or detectors,
+and the placement mechanisms vary. The exact specifications on the type of
+specialised short channel optodes is beyond the scope of the BIDS specification
+and this information can be stored within the SNIRF file (for example, in the `sourcePower` field).
```

Comment:
This wording is odd. "It's beyond the scope of the BIDS specs _and_ this info can be stored..." Could you say that it's beyond the scope of the BIDS specs but the info should still be stored within the SNIRF file?

Reply:
I have reworded. Hopefully this is clearer now.